service/src/test/java/de/adorsys/psd2/sandbox/portal/testdata/TestDataServiceTest.java

```diff
@@ -1,102 +1,124 @@
 package de.adorsys.psd2.sandbox.portal.testdata;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 import de.adorsys.psd2.sandbox.portal.testdata.domain.Account;
 import de.adorsys.psd2.sandbox.portal.testdata.domain.Transaction;
-import java.math.BigDecimal;
-import java.time.LocalDate;
-import java.util.Currency;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
 import org.junit.Before;
 import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 
 public class TestDataServiceTest {
 
-  private static final String ACCOUNT_ID_GIRO = TestDataService.ACCOUNT_ID_GIRO;
-  private static final String IBAN_GIRO = TestDataService.IBAN_GIRO;
-  private static final String ACCOUNT_ID_SAVINGS = TestDataService.ACCOUNT_ID_SAVINGS;
-  private static final String IBAN_SAVINGS = TestDataService.IBAN_SAVINGS;
-
   private TestDataService testDataService;
 
+  @Autowired
+  public TestDataServiceTest() {
+    this.testDataService = new TestDataService();
```

Comment:
Same as with the other constructor injections.

Reply:
Done

Reply:
No constructor injection in tests - my fault.